### DISCUSSION TITLE
Is it possible to Zoom Turtle Window ?

### DISCUSSION THREAD

**Discussion by ShivashishPachauri** [Akascape/TkDial#4] [Show and tell]
Hey Akascape, Thanks for your great contribution in creating tkdial. I have one more doubt but regarding Turtle Python. I saw your oscilioscope where you used Turtle Graphics, so I thought you can help.

Here Im trying to real time draw the geo coordinates of city Delhi using turtle. Function ParsePacket() returns data array which have randomly generated geo-coordinated of Delhi .

 Problem:
I can  see that my turtle pointer actually works correctly but change in value of geocoordinates are very small and I want to know if I can zoom my turtle window and can clearly see the movements.  Can you help me in Zooming the window?

import random
import tkinter as Tkinter
import threading
import time
import turtle

class GUI():
    def ParsePacket():
        n = random.random()
        latitude= 28+n
        longitude=77+n
        
        data =[latitude,longitude]   #random Latitude and Longitude Generator of New Delhi
        return data

def update():
    while True:
        z = GUI.ParsePacket()
        x=z[0]
        y=z[1]
        print(x,y)
        tracy.goto(x,y)
        tracy.pendown()
        

if __name__ == "__main__":
   
    root = Tkinter.Tk()
    START_WIDTH = 700 
    START_HEIGHT = 700 

    frame = Tkinter.Frame(root, width=START_WIDTH, height=START_HEIGHT)
    frame.grid_rowconfigure(0, weight=1)
    frame.grid_columnconfigure(0, weight=1)
    canvas = Tkinter.Canvas(frame, width=START_WIDTH, height=START_HEIGHT,
                            scrollregion=(0, 0, START_WIDTH, START_HEIGHT))

    canvas.grid(row=0, column=0, sticky=Tkinter.N+Tkinter.S+Tkinter.E+Tkinter.W)
    frame.pack()
    tracy = turtle.RawTurtle(canvas, visible=False)
    tracy.penup()
    tracy.goto(50.126,8.605)

    tracy.showturtle()
    #setup( width = 200, height = 200, startx = None, starty = None)
    
    threading.Thread(target=update).start() 
    root.mainloop()

Im also attaching zip file of code, in case above code does not work. 
[Turtle_tracker_by_threading2.zip](https://github.com/Akascape/TkDial/files/11156839/Turtle_tracker_by_threading2.zip)

**Comment by Akascape** [comment]
@ShivashishPachauri 
You can use `turtlesize` to zoom inside the turtle screen and `pensize` to change the line width.

```python
tracy.turtlesize(5) 
tracy.pensize(5)  
```
And please try to put your code properly:
````
```python

# your code here

```
````
